src/CMakeLists.txt

```diff
@@ -41,11 +39,13 @@ set(malos_src
   driver_lirc.cpp driver_lirc.h
   driver_servo.cpp driver_servo.h
   driver_gpio.cpp driver_gpio.h
-  ${MALOS_PROTO_SRC} ${MALOS_PROTO_HEADER}
 )
 
+link_directories(${CMAKE_BINARY_DIR}/protocol-buffers/matrixlabs)
+
 add_executable(malos ${malos_src})
   set_property(TARGET malos PROPERTY CXX_STANDARD 11)
+  target_link_libraries(malos build_protos)
```

Comment:
matrix_labs_protobuf_lib is a better name. "build_protos" sounds like a command.
"build_protos" is most likely inherited because the CMakeLists.txt of the protobufs was only used to check for correctness.